core/server/data/migrations/versions/1.25/2-demo-post.js

```diff
@@ -0,0 +1,55 @@
+const _ = require('lodash');
+const common = require('../../../../lib/common');
+const models = require('../../../../models');
+const fixtures = require('../../../../data/schema/fixtures');
+const message1 = 'Adding demo post.';
+const message2 = 'Added demo post.';
+const message3 = 'Skip: Adding demo post.';
```

Comment:
It would be nice to change this to mention why it was skipped. "Skipped adding demo post - slug already in use"